csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -2086,7 +2100,7 @@ A program satisfies the "CSAF modifier" conformance profile if the program fulfi
 The program:
 
 * satisfies the "CSAF post-processor" conformance profile.
-* adds, deletes or modifies at least one property or object or value of a property.
+* adds, deletes or modifies at least one property or object or value of a property or item of an array.
```

Comment:
I thought about the granularities a modifier might target. When mentioning properties of objects, items of arrays do match. In case e.g. filtering (removing) an item from an array cascades to removal of the whole array to satisfy the schema cardinality constraints, the array level modification would follow.
I will think about it ...

Reply:
> When mentioning properties of objects, items of arrays do match.
This makes totally sense. But weren't we talking about properties OR objects?

Adding an item into an array is basically modifying it ;-) 
We could also use:

> adds, deletes or modifies at least one property, array, object or value of a property or item of an array

I will think about it again...

Reply:
Yes, that:
> adds, deletes or modifies at least one property, array, object or value of a property or item of an array
makes sense to me.

Reply:
I think this is resolved now.